PhysicsTools/PatAlgos/python/slimming/puppiForMET_cff.py

```diff
@@ -21,52 +18,34 @@ def makePuppies( process ):
                                            pdgId = cms.vint32(-11,11,-13,13),
                                            )
     task.add(process.pfLeptonsPUPPET)
-
     addToProcessAndTask('puppiNoLep', process.puppi.clone(), process, task)
     process.puppiNoLep.candName = cms.InputTag('pfNoLepPUPPI') 
-    process.puppiMerged = cms.EDProducer("CandViewMerger",src = cms.VInputTag( 'puppiNoLep','pfLeptonsPUPPET'))
-    task.add(process.puppiMerged)
-    process.load('CommonTools.PileupAlgos.PhotonPuppi_cff')
-    task.add(process.puppiPhoton)
-    addToProcessAndTask('puppiForMET', process.puppiPhoton.clone(), process, task)
-    #Line below replaces reference linking wiht delta R matching because the puppi references after merging are not consistent with those of the original PF collection
-    process.puppiForMET.useRefs          = False
-    #Line below points puppi MET to puppi no lepton which increases the response
-    process.puppiForMET.puppiCandName    = 'puppiMerged'
-
+    addToProcessAndTask('puppiNoLepPhoton', process.puppiNoLep.clone(), process, task)
```

Comment:
would a puppiNoLepModPhoton or puppiNoLepModNeutrals or Mod -> Safe or Protect be a bit more intuitive?